### ISSUE TITLE
Inclusion of a Reference Text for biomolecules 

### ISSUE THREAD

**Issue by julioarvellos** [BradyAJohnston/MolecularNodes#628]
Sorry for the title, I'm not sure how to describe the idea there. I'd like to suggest the inclusion of a Text block that one could access through MN directly (not sure if this is possible) using the Text Editor. This text would contain a few references of PDB/Alphafold codes for common biomolecules that people usually use in different scenes (like IgG, transmembrane proteins, porous, fibrils, ribosomes etc.).

The idea is to give the users additional help by providing some quick starting examples so one could easily pick a code and import the structure, without the extra effort of (re)searching for a specific file in the data banks. 

Below is an example of a text structure that I asked Chat GPT to create:

```
# Date: 02/10/2024

# List of Biomolecules:

# Globular Proteins
| Code      | Database      | Biomolecule Name         |
|-----------|---------------|--------------------------|
| 1A4U      | PDB           | Hemoglobin               |
| 1D66      | PDB           | Lysozyme Enzyme          |
| Q9Y4L1    | AlphaFold     | Beta-glucosidase Enzyme  |

# Fibrous Proteins
| Code      | Database      | Biomolecule Name         |
|-----------|---------------|--------------------------|
| 1SYW      | PDB           | Collagen Type I          |

# DNA / RNA
| Code      | Database      | Biomolecule Name         |
|-----------|---------------|--------------------------|
| 1BNA      | PDB           | B-DNA Structure          |
| 6KR9      | PDB           | Ribosomal RNA            |

# Metalloproteins
| Code      | Database      | Biomolecule Name         |
|-----------|---------------|--------------------------|
| 4HHB      | PDB           | Deoxygenated Hemoglobin  |
| 1COA      | PDB           | Cytochrome c Oxidase     |

# Transmembrane Protein
| Code      | Database      | Biomolecule Name         |
|-----------|---------------|--------------------------|
| 7K8T      | AlphaFold     | SARS-CoV-2 Spike Protein |

```

This is how it would look inside the Text Editor:
![Screenshot_3](https://github.com/user-attachments/assets/a307e7aa-e77b-4f96-a108-26dca405cc04)

This isn't supposed to be an extensive list of biomolecules, but rather a small list of common ones - what 'common' means will certainly depend, but I believe we can discuss this list with others in the Discord server.

The idea is also based on my experience in the past, as I had some trouble finding protein structures when I once started working with Blender.

Some PROs:
- It is a light option compared to creating a library or packing structures directly inside MN (the alternative is doing nothing haha);
- The text can be loaded only when actively enabled/requested - this avoids the creation of this data block in each blender file;
- The structures can be linked to future tutorials of MN;
- The text block could have a header with more information about MN (website, version, etc.)

Some CONS:
- I'm not sure how the DB handle the codes or if they can change in the future, but the community can help checking if anything goes wrong - also, with a small list this isn't a big problem;

I'll let you fill the CONS list because I'm only thinking about the positive things at the moment...

**Comment by BradyAJohnston** [comment]
I like the thinking here. I think the ideal form that this would take is using Blender's asset library, to provide a selection of common assets for different domains (blood plasma, human cytoplasm etc) allowing for quick drag and drop import.

There are various improvements that are going on for Blender 4.3 and 4.4 that will better enable such a workflow. Before that point however, I would be happy to include some more written documentation that is providing an overview of common proteins that users might want to import. 

Such written documentation would be better suited to live on the documentation website (https://bradyajohnston.github.io/MolecularNodes/) rather than in any kind of bundled text file though. It's easier to update and track, can be updated independently outside of released MN versions etc. 

Compiling and curating such a list is no small task however. It's not something that I would be able to take on, but if you wanted to try and lead an effort the including it in the online documentation would be great.

**Comment by julioarvellos** [comment]
I agree with you that the documentation should be the first place the users should find examples more about MN and I could help with some things, sure. Do you have any guidelines or tools you recommend to use? My English will certainly need review, but I can get some initial help with an AI.

Regarding the assets, I was thinking about something different from an asset library. The idea is to allow the (noob) users to find a quick 'start sample' to help them create their scenes but also without adding too much extra work for this. But as you mentioned, maybe in the future it will be easier to generate the asset library using a list from the documentation... so let's improve the docs.
